apps/dev/pages/api/auth/[...nextauth].ts

```diff
@@ -193,6 +194,12 @@ export const authOptions: NextAuthOptions = {
       clientId: process.env.PATREON_ID,
       clientSecret: process.env.PATREON_SECRET,
     }),
+    AsgardeoProvider({
+      organization: process.env.ASGARDEO_ORGANIZATION,
```

Comment:
```suggestion
      issuer: process.env.ASGARDEO_ISSUER,
```

Let's use `issuer` which is according to the spec, and consistent with the other providers

Reply:
+1